schemas/com.snowplowanalytics.snowplow.enrichments/api_request_enrichment_config/jsonschema/1-0-0

```diff
@@ -0,0 +1,167 @@
+{
+	"$schema": "http://json-schema.org/schema#",
+	"description": "Schema for API Request Enrichment configuration",
+	"self": {
+		"vendor": "com.snowplowanalytics.snowplow.enrichments",
+		"name": "api_request_enrichment_config",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+
+	"type": "object",
+	"properties": {
+		"vendor": {
+			"type": "string"
+		},
+		"name": {
+			"type": "string"
+		},
+		"enabled": {
+			"type": "boolean"
+		},
+
+		"parameters": {
+			"type": "object",
+			"properties": {
+				"inputs": {
+					"type": "array",
+					"items": {
+						"type": "object",
+						"properties": {
+							"key": {
+								"type": "string",
+								"pattern": "^[A-Za-z0-9_-]+$"
+							},
+							"pojo": {
+								"type": "object",
+								"properties": {
+									"field": {
+										"type": "string"
+									}
+								},
+								"additionalProperties": false
+							},
+							"json": {
+								"type": "object",
+								"properties": {
+									"field": {
+										"type": "string",
+										"enum": ["unstruct_event", "contexts", "derived_contexts"]
+									},
+								  "schemaCriterion": {
+									  "type": "string",
+									  "pattern": "^iglu:[a-zA-Z0-9-_.]+/[a-zA-Z0-9-_]+/[a-zA-Z0-9-_]+/[0-9]+-([0-9]+|\\*)-([0-9]+|\\*)$"
+								  },
+								  "jsonPath": {
+									  "type": "string"
+								  }
+								},
+								"additionalProperties": false
+							}
+						},
+						"additionalProperties": false,
+						"minProperties": 2,
+						"maxProperties": 2,
+						"required": ["key"]
+					}
+				},
+				"api": {
+					"type": "object",
+					"minProperties": 1,
+					"maxProperties": 1,
+					"properties": {
+						"http": {
+							"type": "object",
+							"properties": {
+								"method": {
+									"type": "string",
+									"enum": ["GET", "POST", "PUT", "DELETE"]
```

Comment:
1) Sure - but if a `GET` deletes data, that's the responsibility of the API author; they can't say we've done anything wrong on our side 2) I'm referring to a bad actor - e.g. a disgruntled analyst who has the ability to update Snowplow enrichments 3) sure - but if somebody has implemented `DELETE` correctly, then it will delete data 4) see 2)

The wider point is: HTTP methods which exist to delete data shouldn't be supported in an enrichment which is meant to be read-only.